src/app/portlets/content-types/fields/content-type-fields-variables/add-variable-form/add-variable-form.component.html

```diff
@@ -7,5 +7,5 @@
         <label class="form__label">{{ messages['contenttypes.field.variables.value_header.label'] }}</label>
         <input pInputText type="text" formControlName="value"/>
     </div>
-    <button pButton secondary tiny [label]="messages['contenttypes.field.variables.add_button.label']" (click)="formEl.ngSubmit" [disabled]="!form.valid"></button>
+    <button type="submit" pButton secondary tiny [label]="messages['contenttypes.field.variables.add_button.label']" [disabled]="!form.valid"></button>
```

Comment:
You don't need to bind the click if the button type is submit and you want to submit the form.